django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -15,6 +15,7 @@ Requires core.js and SelectBox.js.
             const from_box = document.getElementById(field_id);
             from_box.id += '_from'; // change its ID
             from_box.className = 'filtered';
+            from_box.setAttribute('aria-label', 'Available permissions');
```

Comment:
We need to make sure we translate any texts and this is also a general widget and not specific to permissions